web/src/components/Preferences.jsx

```diff
@@ -242,19 +242,14 @@ const WebPushEnabled = () => {
     await prefs.setWebPushEnabled(ev.target.value);
   };
 
-  // while loading
-  if (defaultEnabled == null) {
-    return null;
-  }
-
   if (!notifier.pushPossible()) {
     return null;
   }
 
   return (
     <Pref labelId={labelId} title={t("prefs_notifications_web_push_title")} description={t("prefs_notifications_web_push_description")}>
       <FormControl fullWidth variant="standard" sx={{ m: 1 }}>
-        <Select value={defaultEnabled} onChange={handleChange} aria-labelledby={labelId}>
+        <Select value={defaultEnabled ?? false} onChange={handleChange} aria-labelledby={labelId}>
```

Comment:
Won't this make the UI show "Disabled" when loading, even if the preference is true?

Reply:
Yes, but only very shortly. I have to figure the whole loading thing out overall anyway. It's very ugly to have a two DOM changes in succession that change the height of components. It's very choppy looking.

It's just personal preference to do it this way. Plus, false is the default.